Cython/Utils.py

```diff
@@ -454,6 +455,16 @@ def str_to_number(value):
     return -value if is_neg else value
 
 
+def strip_l_py2(value_str):
```

Comment:
`l` is easy to confuse with `1` and `I` in many fonts, especially `1` in the context of numbers. Maybe `strip_py2_long_suffix`?